tests/kafkatest/services/kafka/kafka.py

```diff
@@ -508,4 +508,20 @@ def get_offset_shell(self, topic, partitions, max_wait_ms, offsets, time):
         for line in node.account.ssh_capture(cmd):
             output += line
         self.logger.debug(output)
+        return output
+
+    def get_simple_consumer_shell(self, topic, partition=0):
```

Comment:
Ok I think I see your reasoning, but I still disagree, so let's see if I can convince you :) Even though it's not necessarily a long-running thing, I still think it makes sense to do this as a standalone service. Here are a couple arguments in favor that come to mind:

- Services don't have to be long-running to qualify as services
- It seems odd to me to have a this KafkaService method since it essentially runs a client process
- It looks like `SimpleConsumerShell` *could* be used as a long running service even though that's not how you're using it in this particular test. It's arguably as much of a service as `ConsoleConsumer`
- `SimpleConsumerShell` has a bunch of options you're not using here. As a standalone Service class, it's much easier to extend and run with different configurations.
- By using it as a service, ducktape can automatically collect the generated logs for later debugging
- By using it as a service, we can automatically take advantage of cleanup portion of the lifecycle

Thoughts?